code/game/gamemodes/devil/devilinfo.dm

```diff
@@ -435,5 +435,6 @@ var/global/list/lawlorify = list (
 	soulCounter = SOULVALUE
 	var/obj/screen/devil/soul_counter/K = new()
 	K.update()
+	var/mob/living/carbon/D = new()
```

Comment:
Oh right. current is a mob/living, whereas handle_devil is defined under mob/living/carbon

Just move the handle_devil proc